components/attribute_types/DateTimeAttributeType.js

```diff
@@ -0,0 +1,60 @@
+import AttributeType from '@/components/attribute_types/AttributeType';
+import { useFormikContext, useField } from "formik";
+import { useState, useEffect } from 'react';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+import moment from 'moment'; 
+
+const DateTimeAttributeReadOnlyComponent = ({ text }) => {
+    // Parse the text (selectedDate) using Date object
+    const parsedDate = new Date(text);
+    // extract the text from the html elements
+    const extractTextFromHTML = (html) => {
```

Comment:
I don't think the `extractTextFromHTML` function is not needed anymore, because I only used this function for the Rich Text Attribute. This function removes the `html elements` in the string. In this case, there are no html elements in the text so you can remove this function. 

You can try this instead.

```suggestion
    const DateTimeAttributeReadOnlyComponent = ({ text }) => {
      const parsedDate = new Date(text);
      const formattedDate = moment(parsedDate).format('MMM. D, YYYY - hh:mm A');
      return formattedDate;
    }
```